src/app/ui/editor/select_box_state.h

```diff
@@ -26,6 +27,7 @@ namespace app {
     // Called each time the selected box is modified (e.g. rulers are
     // moved).
     virtual void onChangeRectangle(const gfx::Rect& rect) { }
+    virtual void onChangePadding(const gfx::SizeT<int>& padding) { }
```

Comment:
```suggestion
    virtual void onChangePadding(const gfx::Size& padding) { }
```
Just `gfx::Size` with a `#include "gfx/fwd.h"` above.